components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -249,6 +249,8 @@ public abstract class FrameworkConstants {
     public static final String ERROR_DESCRIPTION_AUTH_CONTEXT_NULL = "authentication.context.null.description";
     public static final String ERROR_STATUS_AUTH_FLOW_TIMEOUT = "authentication.flow.timeout";
     public static final String ERROR_DESCRIPTION_AUTH_FLOW_TIMEOUT = "authentication.flow.timeout.description";
+    public static final String ERROR_DESCRIPTION_ACCESS_DENIED = "authentication.flow.access.denied.description";
```

Comment:
Looking at the code this seems to be a error status and `ERROR_DESCRIPTION_ACCESS_DISABLED` is meant to be its description